.github/workflows/lint.yaml

```
@@ -21,10 +21,10 @@ jobs:
       - name: Install dependencies
         run: |
           pip install --upgrade pip
-          pip install .[dev,github-actions]
+          pip install .[dev,dependencies,github-actions]
```

これdependencies要りましたっけ？ feature/v2-torchでは入れなくて動いているので，削除下さい．
```suggestion
          pip install .[dev,github-actions]
```